openshift/templates/api/api-dc-template-v2.yaml

```diff
@@ -169,7 +169,7 @@ objects:
             resources:
               limits:
                 cpu: 150m
-                memory: 1Gi
+                memory: 500Mi
```

Comment:
Is there a way to test this before making this change? 

Run the application and do a 'top' to see the resources it is consuming. Add a 15% buffer and see if 500MB is sufficient? 

The most resource intensive part of the application is pdf creation (for now). May be do two pdf creation simultaneously and see how much resource it consumes? It is not concurrent but we can measure time.  
Measuring the time difference - how long pdf creation takes between 1GB and 500MB? 

If it take too long by limiting resources the P.O should be aware. It affects the UI/UX  directly.

Reply:
I'm currently making these changes to allow for more containers to be spun up.

These changes are currently being used in DEV, so if you're seeing performance issues I need to know ASAP, as they've been live in the environment since Friday morning last week.

Reply:
did some basic testing. For our current use case it is within the limit.